Anguandia/quickcredit
# Load search page for individual item-targeting menu items

#### What does this PR do?
Load search page for specified menu item target

#### Description of Task to be completed?
On selection of admin menu items targeting a specific loan or user object,
load the item search/select page to select the target item. Breakdown:
 - Anchor menu items targeting specific items(approve loan, verify user,
    debit loan, loan details, and client details) (template.js file) to the
    search page(client.html)
 - Append a query parameter to these anchor hrefs with the page title as
    key and target class for selective loading as value; rename these classes
    to reflect the page purpose(to reconstitute the same)
- In the search page(client.js file), anchor each collection item and search button to
    the target's detail page(detail.html)
- Append current URL's query string to each anchor href
 - Add decodeQuery function to read URL query string, constitute page
    title and purpose and call the selective page loading function (showPage)
    with the required class from query value
 - Modify detail page to trigger decodeQuery onload

#### How should this be manually tested?
- Visit https://anguandia.github.io/quickCredit/ or clone this repo and
    open the index, signup or signin.html file
 - Signing as admin
 - From the menu on the left, select any of verify client, approve loan,
    debit loan, loan details, and client details items
 - From the resulting search page, click on an item or enter an id in the
    search box and click select
 - Verify that the page rendered is the page requested

#### Any background context you want to provide?
Currently, clicking on the specific-item-targeting menu items would directly render the sample detail page without an option to specify the target

#### What are the relevant pivotal tracker stories?
[fixes #165489723 #165489637 #165489658 #165489749 #165489571]
